src/test/java/ch/uzh/ifi/hase/soprafs22/service/UserServiceIntegrationTest.java

```diff
@@ -93,6 +98,7 @@ void createUser_duplicateUsername_throwsException() {
     testUser2.setStatus(UserStatus.ONLINE);
     testUser2.setToken("b");
     testUser2.setPassword("1234");
+    testUser2.setBirthday(new Date());
```

Comment:
Why do you set the birthday as today (would make more sense to have him be older than 18)

Reply:
In all of them it doesn't really make a difference, as it is not what we are testing here